scripts/funcTests/runFuncTests.sh

```diff
@@ -81,7 +79,7 @@ do
 
 	echo "Channel is: $Channel"
 	echo "Version is: $Version"
-	scripts/funcTests/dotnet-install.sh -i cli -c $Channel -v $Version -nopath
```

Comment:
I didn't add `-Quality Daily` since we're passing Version. According to [doc](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#:~:text=%2DVersion%7C%2D%2Dversion%20%3CVERSION%3E) if we pass version then both `channel` and `-Quality Daily` are ignored. Whatever happened with [#971](https://github.com/NuGet/Client.Engineering/issues/971) issue was temporary bug that time. I don't see it on my local, that means I can remove `-Quality Daily` below line too, please let me know if it's necessary.
https://github.com/NuGet/NuGet.Client/blob/aa9721f527cb8f25d170d22b8ebaf5e1906cb7fb/build/common.ps1#L258

Reply:
[14:42:37 +0]   [BEGIN] Installing .NET CLI
[14:42:38 +0]   The version of SDK should be installed is : 5.0.200-servicing.21120.4
[14:42:38 +0]   Probing folder : C:\xxxxxx\NuGet.Client\cli\sdk\5.0.200-servicing.21120.4
[14:42:38 +0]   C:\xxxxxx\NuGet.Client\cli\dotnet-install.ps1 -InstallDir C:\xxxxxx\NuGet.Client\cli -Version 5.0.200-servicing.21120.4 -Architecture x64 -NoPath
dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.

dotnet-install: Downloading primary link https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.200-servicing.21120.4/dotnet-sdk-5.0.200-servicing.21120.4-win-x64.zip
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.200-servicing.21120.4/dotnet-sdk-5.0.200-servicing.21120.4-win-x64.zip
dotnet-install: Binaries of dotnet can be found in C:\xxxxxx\NuGet.Client\cli\
dotnet-install: Note that the script does not resolve dependencies during installation.
dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies
dotnet-install: Installation finished

Reply:
Please check how it looks after removing `-Channel ` and `-Quality Daily` from build/common.ps1

[17:42:18 +1]   Downloading .NET CLI 'release/5.0.2xx:5.0.200-servicing.21120.4'
[17:42:19 +1]   The version of SDK should be installed is : 5.0.200-servicing.21120.4
[17:42:19 +0]   Probing folder : C:xxxxxx\NuGet.Client\cli\sdk\5.0.200-servicing.21120.4
[17:42:19 +0]  ` C:xxxxxx\NuGet.Client\cli\dotnet-install.ps1 -InstallDir C:xxxxxx\NuGet.Client\cli -Version 5.0.200-servicing.21120.4 -Architecture x64 -NoPath`
dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.

dotnet-install: Downloading primary link https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.200-servicing.21120.4/dotnet-sdk-5.0.200-servicing.21120.4-win-x64.zip
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.200-servicing.21120.4/dotnet-sdk-5.0.200-servicing.21120.4-win-x64.zip
dotnet-install: Binaries of dotnet can be found in C:xxxxxx\NuGet.Client\cli\
dotnet-install: Note that the script does not resolve dependencies during installation.
dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies
dotnet-install: Installation finished

Reply:
We need to be able to download the latest verison from a channel. 

In fact, we usually prefer to do that. 

We tied to the latest version by necessity at some point during the net5 development and never got around to bring it back. @heng-liu is working on the 6.0.

Reply:
Considering there's parallel effort going on by Heng, I removed other extra changes, now kept only necessary changes.
Please check now.